src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3540,6 +3540,19 @@
     public static final int COMMENT_CONTENT =
             GeneratedJavaTokenTypes.COMMENT_CONTENT;
 
+    /**
+     * Java 14 Pattern definition.
+     *
+     * <pre>
+     * +--PATTERN_DEF
+     *          |
+     *          +VARIABLE_DEF
```

Comment:
Please put a bit more nodes to see  it in surrounding/expression

Reply:
Done.

Reply:
please do this with code, good example - https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CTOR_DEF

One day we will make all docs like this. It very useful, as nobody remember how code converted to AST.

Reply:
Fixed

Reply:
I do not see real java code. please recheck that you pushed changes.

Reply:
I did not push, my apologies. it is done now.